src/test/java/com/sngular/kloadgen/extractor/extractors/AvroExtractorTest.java

```diff
@@ -38,17 +29,30 @@ void testFlatPropertiesEmbeddedAvros() throws Exception {
                   FieldValueMapping.builder().fieldName("timestamp").fieldType("long").fieldValueList("").required(true).isAncestorRequired(true).build()
               );
   }
+  @Test
+  @DisplayName("Should extract Embedded Record")
+  void testFlatPropertiesEmbeddedAvros() throws Exception {
+
+    final String testFile = fileHelper.getContent("/avro-files/embedded-avros-example-test.avsc");
```

Comment:
I think you are using this file several times in the test, so maybe is a good idea to create a constant to hold that value